db-builder/src/main/java/com/yahoo/covid19/database/models/Places.java

```diff
@@ -164,13 +159,10 @@ public Object getId() {
 
     @Override
     public String getTableName() {
-        return type == null ? "" : type.name();
+        return TABLE_NAME;
     }
 
     @Override
     public void getForiegnKeyFields(Map<String, Insertable> foreignKeyMap) {
-        if (type.equals(JoinTableNames.Supername)) {
```

Comment:
Can we remove this entire function from INsertable?

Reply:
healthRecord uses this function to populate lat, lon, wikiId